nilearn/signal.py

```diff
@@ -833,12 +848,26 @@ def _censor_signals(signals, confounds, sample_mask):
     return signals, confounds
 
 
-def _interpolate_volumes(volumes, sample_mask, t_r):
+def _interpolate_volumes(volumes, sample_mask, t_r, extrapolate):
     """Interpolate censored volumes in signals/confounds."""
+    if extrapolate:
+        extrapolate_default = (
+            "By default the cubic spline interpolator extrapolates "
+            "the out-of-bounds censored volumes in the data run. This "
+            "can lead to undesired filtered signal results. In future "
+            "releases, the default strategy will be not to extrapolate "
```

Comment:
Starting from release 0.13,...

Reply:
Fixed in 9f2c5258faf450b41659a6f5430deac1fa68efc5